implementations/lighthouse/app/Models/User.php

```diff
@@ -9,7 +9,10 @@
  * @property string $email
  *
  * Attributes
+ * @property string|null $name
  * @property int|null $totalProductsCreated
+ * @property int $yearsOfEmployment
+ * @property int|null $averageProductsCreatedPerYear
```

Comment:
This would only be available as a property if the method returned an accessor: https://laravel.com/docs/10.x/eloquent-mutators#defining-an-accessor

Alternatively, you can omit this annotation and annotate the field in `User.graphql` with `@method`.